lib/sensu-plugins-zfs/version.rb

```diff
@@ -2,7 +2,7 @@ module SensuPluginsZFS
   module Version
     MAJOR = 2
     MINOR = 0
```

Comment:
As this change adds new functionality, you should bump the minor version.

Reply:
we manage the versioning post acceptance anyways